netket/driver/vmc_common.py

```diff
@@ -14,7 +14,7 @@
 
 
 def info(obj, depth=None):
-    if hasattr(obj, "info"):
+    if hasattr(obj, "info()"):
```

Comment:
```suggestion
    if hasattr(obj, "info") and callable(obj.info):
```

Reply:
Sorry about that. My hasattr syntax doesn't really work. Using just `hasattr(obj, "info")` was causing an error because `preconditioner` has an attribute "info", which is not callable. Your change fixes it.